src/pages/meetup/[id].astro

```diff
@@ -1,22 +1,30 @@
 ---
 import LayoutSingleMeetup from "@layouts/LayoutSingleMeetup.astro";
 import MeetupSingle from "@components/MeetupSingle.astro";
-import { getEvent, loadEvents } from "@utils/api";
+import eventsResponse from "src/data/meetups-raw.json";
 import EventJSONLD from "@components/EventJSONLD.astro";
 
 const props = Astro.params;
 
 export async function getStaticPaths() {
-  const events = await loadEvents();
-  return events.data.map((event) => {
+  return eventsResponse.data.map((event) => {
     return {
       params: {
         id: event.id.toString(),
       },
     };
   });
 }
-const meetup = await getEvent(props.id);
+const getEvent = (id: string | number) => {
+  let event = eventsResponse.data.find((ev) => ev.id == id);
+
+  if (event === null) {
```

Comment:
Is there a reason why we explicitly check for `null` here?
I know that from JSON we should not get `null` and not `undefined`, but curious whether there is a reason for the explicit `null` instead of a `falsy` check

Reply:
tI kept this api function as-is when i migrated them here. as is the case for all the other ones. i guess the null check was cuz it was expecting data directly from directus. but nicely spotted! should probably also do a clean up of these things before merging everything

Reply:
@cedpoilly iirc, if there's a page with a bad meetup id, astro would give null back, hence the need for this check. This can be improve however. I'm not totally sure this is the best way to implement. Perhaps a falsy check like you said might be better. Shouldn't be optimized in this PR.